ast/functions.go

```diff
@@ -338,26 +338,40 @@ func (n *FuncTrimExpr) IsStatic() bool {
 	return n.Str.IsStatic() && n.RemStr.IsStatic()
 }
 
-// DateArithType is type for DateArith option.
+// DateArithType is type for DateArith type.
 type DateArithType byte
 
 const (
+	// AddDate is to run adddate function option.
+	// See: https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_adddate
+	AddDate DateArithType = iota + 1
 	// DateAdd is to run date_add function option.
 	// See: https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_date-add
-	DateAdd DateArithType = iota + 1
+	DateAdd
```

Comment:
AddDate and DateAdd seem confused. use only one. we can use data_add and date_sub inner, no need to case other synonyms.